PlaywrightSolar/Tests/ProductDetailsTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using PlaywrightSolar.PageObjects;
+
+namespace PlaywrightSolar.Tests;
+
+internal class ProductDetailsTest : UiTestFixture
+{
+    private InvertersPage _invertersPage;
+    
+    [SetUp]
+    public void SetupSolarTechnologyShopPage()
```

Comment:
no need to specify which exactly pages are setup in the method name. This causes confusion like in this case - name is SetupSolarTechnologyShopPage which creates InvertersPage
just public void Setup() is enough